Dockerfile

```diff
@@ -51,15 +52,16 @@ RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.n
 
 VOLUME /data
 
-RUN  mkdir -p /etc/fluent/config.d
+RUN  mkdir -p /etc/fluent/configs.d ${HOME}/forwarder-example
+COPY data/ ${HOME}/forwarder-example/
```

Comment:
@t0ffel No, it isn't used in the simple case.  I left it there for your use case.